students/vkis/lesson02/part123.py

```diff
@@ -0,0 +1,39 @@
+# function that draws a static 2x2 grid
+# function name is gird()
+
+# individual graphic elements
+plus = "+ "
+minus = "- "
+vertical = "| "
+space = "  "
+
+# full grid
+def grid(row, col, size):
+	# box structure
+	top = plus + minus * size
+	side = vertical + minus * size
+	
+	i = 1
+	while i <= row:							#print bunch of rows
```

Comment:
I think this might be better written via a `for` loop rather than an incrementing `while` loop.  Can you try to convert all your `while` loops in this .py file to for?